src/frontend/hooks/useSlåSammenPerioder.ts

```diff
@@ -0,0 +1,27 @@
+import { useHttp } from '@navikt/familie-http';
+import { type Ressurs, RessursStatus } from '@navikt/familie-typer';
+import { useState } from 'react';
+
+export const useSlåSammenPerioder = (behandlingId: string, skalSammenslaa: boolean) => {
+    const { request } = useHttp();
+    const [feilmelding, settFeilmelding] = useState<string>();
+
+    const slåSammenPerioder = async () => {
+        const response: Ressurs<boolean> = await request<void, boolean>({
+            method: 'POST',
+            url: `/familie-tilbake/api/perioder/slaa-sammen-perioder/${behandlingId}?skalSammenslaa=${skalSammenslaa}`,
```

Comment:
Mulig jeg er litt naiv, men synes "skalSammenslaa=true/false" er litt ulogisk.

Kan dette deles opp i to endepunkter? 
- Slå sammen perioder
- Angre slå sammen perioder

Reply:
Enig, disse er nå delt opp i to endepunkter.